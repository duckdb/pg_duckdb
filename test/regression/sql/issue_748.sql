SET duckdb.force_execution = TRUE;

CREATE SCHEMA selinto_schema;

CREATE TABLE selinto_schema.tbl_nodata4 (a) AS

EXECUTE data_sel WITH NO DATA;

DROP SCHEMA selinto_schema CASCADE;
