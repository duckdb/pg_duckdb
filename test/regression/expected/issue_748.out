SET duckdb.force_execution = TRUE;
CREATE SCHEMA selinto_schema;
CREATE TABLE selinto_schema.tbl_nodata4 (a) AS
EXECUTE data_sel WITH NO DATA;
ERROR:  prepared statement "data_sel" does not exist
DROP SCHEMA selinto_schema CASCADE;
